airbyte-integrations/connectors/source-pipedrive/source_pipedrive/streams.py

```diff
@@ -136,13 +142,25 @@ class Organizations(PipedriveStream):
     """
 
 
+class OrganizationFields(PipedriveStream):
+    """https://developers.pipedrive.com/docs/api/v1/OrganizationFields#getOrganizationFields"""
+
+    primary_key = None
+
+
 class Persons(PipedriveStream):
     """
     API docs: https://developers.pipedrive.com/docs/api/v1/Persons#getPersons,
     retrieved by https://developers.pipedrive.com/docs/api/v1/Recents#getRecents
     """
 
 
+class PersonFields(PipedriveStream):
```

Comment:
This implementation is running as a custom connector in my own infrastructure

Reply:
@orlandoburli I don't think that is the case since as you can see the connector is using **_recents_** only if you provide the **_replication_start_date_** and that is not the case for the **_PersonFields_** stream. 

```
 if self._replication_start_date:
            return "recents"

        class_name = self.__class__.__name__
        return f"{class_name[0].lower()}{class_name[1:]}"
```

Reply:
And how do I know when this field (**replication_start_date**) is used or not?

Reply:
When you open **_source.py_** you can see that there are two arguments **_stream_kwargs_** and **_incremental_kwargs_**. Depending on which argument is passed to the stream it will be treated as incremental or not. In this case PersonFields is not treated as incremental since it has the **_stream_kwargs_** passed.

```
PersonFields(**stream_kwargs)
```